entity-framework/core/miscellaneous/rc1-rc2-upgrade.md

```diff
@@ -1,156 +0,0 @@
----
```

Comment:
missing redirect link

Reply:
This is deleted content, where would you redirect? Would there be any value in that?

Reply:
The guidance is rather than showing them content not found error, we show them most appropriate page which at least keep user in the context. This is heuristic so it may not be always correct page but still better than error page.
For these deleted files, we can just redirect them to upgrade to latest.

Reply:
OK, add redirects to https://docs.microsoft.com/en-us/ef/core/what-is-new which lists all the versions, seems reasonable...